test/mockLightspeedServer/server.ts

```diff
@@ -25,12 +25,12 @@ export default class Server {
     app.get("/", (req, res) => res.send("Lightspeed Mock"));
 
     app.post(`${API_ROOT}/ai/completions`, async (req, res) => {
-      await new Promise((r) => setTimeout(r, 100)); // fake 100ms latency
+      await new Promise((r) => setTimeout(r, 500)); // fake 500ms latency
```

Comment:
Increased the fake latency so that the another inline suggestion provider shows its suggestion first.